README.md

```diff
@@ -1,71 +1,75 @@
+### Successfully running openpilot
 
-> [!NOTE]
-> A notable vehicle not included here is the US-made (VIN does not start with `J`) 2023 Corolla. It does not appear to have ECU Security Key or SecOC steps when replacing the forward camera. If you have one of these US Corollas, stop by the [comma Discord](https://discord.comma.ai). It's exact standing or nature is unknown.
+These cars can run openpilot but are not listed on https://comma.ai/vehicles or [CARS.md](https://github.com/commaai/openpilot/blob/master/docs/CARS.md) because Comma (the company) understandably doesn't want to own the security key hacking process. Follow the [Setup Guide](#setup-guide) below and you'll have it working.
 
-> [!NOTE]
-> Bold vehicles are vehicles we suspect might work with the current key dumping procedures but no one has stopped by and/or attempted on them yet. It is speculation.
+* 2021-2023 RAV4 Prime
+* 2021-2023 Sienna
+* 2020+ Yaris Hybrid (EUDM/JDM/MXDM)
+  * Needs extra steps. See [Support Status Overview](#support-status-overview).
 
-> [!NOTE]
-> Vehicles that are crossed out are supported by openpilot but are not on https://comma.ai/vehicles because they are not fully officially supported. The primary reason stems from the key extraction process being produced from a non-comma.ai process. That said, if you have the key, it will work.
+### May be possible to hack but hasn't been tried
 
-* ~~2021-2023 RAV4 Prime~~
-  * Known as RAV4 PHEV in non-North American markets
-* 2024 RAV4 Prime
-* 2021+ Venza
-* ~~2021-2023 Sienna~~
-* 2024 Sienna
-* **(EUDM/JDM/MXDM) 2021+ Yaris GR**
-* (USDM) 2022+ Corolla Cross (Speculated from TechInfo lookup)
-  * Not applicable to Thailand or Brazil.
-* **(EUDM/JDM/MXDM) 2021+ Yaris Cross Hybrid**
-* ~~(EUDM/JDM/MXDM) 2020+ Yaris Hybrid~~
-* 2022+ Lexus NX (Speculated from TechInfo lookup)
-* 2022+ Tundra (Confirmed in https://github.com/commaai/openpilot/issues/27869#issuecomment-1504046497 )
-* (EUDM) 2022+ Aygo X (Speculated from Toyota Tech EU lookup)
-* 2022+ Lexus LS (Speculated from TechInfo lookup)
-* 2022+ Lexus LX (Speculated from TechInfo lookup)
-* 2023+ Sequoia (Speculated from Being a Tundra With a SUV Body)
-* 2023+ bz4x (Speculated from TechInfo lookup, also probably the same for sister rebranded Subaru Solterra)
-* 2023+ TMC/JP-made Corolla (Speculated from TechInfo lookup)
+If you have one of these cars, please stop by the [comma Discord](https://discord.comma.ai)'s #toyota-security channel - we need more information from people like you.
+
+* 2023 US-made Corolla (VIN does not start with `J`)
+  * It does not appear to have ECU Security Key or SecOC steps when replacing the forward camera. It's exact standing or nature is unknown.
+* 2023 TMC/JP-made Corolla[^3]
   * It is also unknown what form, if any amount of TSK there is on US-made 2023 Corollas. Maybe they just don't do the pairing thing but hardcode a key. No one knows.
```

Comment:
Should this say this instead?
> It is also unknown what form, if any amount of TSK there is on `non-`US-made 2023 Corollas.

Reply:
37 seems to be a stronger assertion than 39. Also, moving 39 would leave no comments for TMC/JP-made Corolla as to why that's considered possible to hack.

Maybe you can come up with a new sentence for US-made Corolla and TMC/JP-made Corolla each, if you can recall the status?

Reply:
We don't really know much about the JP-made corolla. We know the US Corolla is special in that it seems to be the only year + TSS3 combination with no ECU Security Key step. JP Corolla is probably similar to all other TSS3 vehicles.

Reply:
Gotcha. I'll try new sentences and you can correct me